plugins/router/f5/f5.go

```diff
@@ -0,0 +1,555 @@
+package f5
+
+import (
+	"bytes"
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/golang/glog"
+)
+
+type F5Error struct {
+	verb           string
+	url            string
+	httpStatusCode int
+	err            error
+	JsonStatusCode int     `json:"code"`
```

Comment:
`JsonStatusCode` and `Message` are populated from the JSON blob in the HTTP response body whereas `httpStatusCode` is assigned the code in the HTTP response header.

Fair point that there are no current users of `JsonStatusCode`.  I could add validations to check that `JsonStatusCode` matches `httpStatusCode`, but there probably isn't much value in doing so, so I'll just remove `JsonStatusCode`.